Fix pre-commit complains

fridex opened the pull request.

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **fridex**
You can assign the PR to them by writing `/assign @fridex` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=thoth-station%2Fadviser).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/thoth-station/adviser/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["fridex"]} -->

sesheta commented:
<details>
<summary>Pre-Commit Test failed! Click here</summary>

```
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy:attrs~=19.3.
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8:pep8-naming.
[INFO] Initializing environment for https://github.com/mgedmin/check-manifest.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/mgedmin/check-manifest.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for byte-order marker..............................................Passed
Check for case conflicts.................................................Passed
Check docstring is first.................................................Passed
Check JSON...............................................................Passed
Check for broken symlinks................................................Passed
Detect Private Key.......................................................Passed
Check python ast.........................................................Passed
Debug Statements (Python)................................................Passed
pydocstyle...............................................................Failed
- hook id: pydocstyle
- exit code: 1

tests/boots/backports/test_mock.py:103 in public method `test_remove`:
        D400: First line should end with a period (not 'y')
tests/boots/backports/test_mock.py:120 in public method `test_remove_develop`:
        D400: First line should end with a period (not 'y')
tests/boots/backports/test_importlib_metadata.py:103 in public method `test_remove`:
        D400: First line should end with a period (not 'y')
tests/boots/backports/test_importlib_metadata.py:120 in public method `test_remove_develop`:
        D400: First line should end with a period (not 'y')
tests/boots/backports/test_enum34.py:105 in public method `test_remove`:
        D400: First line should end with a period (not 'y')
tests/boots/backports/test_enum34.py:122 in public method `test_remove_develop`:
        D400: First line should end with a period (not 'y')
tests/boots/backports/test_importlib_resources.py:103 in public method `test_remove`:
        D400: First line should end with a period (not 'y')
tests/boots/backports/test_importlib_resources.py:120 in public method `test_remove_develop`:
        D400: First line should end with a period (not 'y')

Check Toml...............................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

thoth/adviser/boots/backports/enum34.py:36:26: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
thoth/adviser/boots/backports/importlib_resources.py:36:38: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
thoth/adviser/boots/backports/mock.py:36:24: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
thoth/adviser/boots/backports/importlib_metadata.py:36:37: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
Found 4 errors in 4 files (checked 70 source files)

black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted /workspace/repo/tests/boots/backports/test_importlib_resources.py
reformatted /workspace/repo/tests/boots/backports/test_mock.py
reformatted /workspace/repo/thoth/adviser/boots/backports/enum34.py
All done! ✨ 🍰 ✨
3 files reformatted, 137 files left unchanged.

flake8...................................................................Passed
check-manifest...........................................................Passed
```
</details>

sesheta commented:
<details>
<summary>Pre-Commit Test failed! Click here</summary>

```
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy:attrs~=19.3.
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8:pep8-naming.
[INFO] Initializing environment for https://github.com/mgedmin/check-manifest.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/mgedmin/check-manifest.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for byte-order marker..............................................Passed
Check for case conflicts.................................................Passed
Check docstring is first.................................................Passed
Check JSON...............................................................Passed
Check for broken symlinks................................................Passed
Detect Private Key.......................................................Passed
Check python ast.........................................................Passed
Debug Statements (Python)................................................Passed
pydocstyle...............................................................Passed
Check Toml...............................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

thoth/adviser/boots/backports/enum34.py:36:26: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
thoth/adviser/boots/backports/importlib_resources.py:36:38: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
thoth/adviser/boots/backports/mock.py:36:24: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
thoth/adviser/boots/backports/importlib_metadata.py:36:37: error: Class cannot subclass 'Boot' (has type 'Any')  [misc]
Found 4 errors in 4 files (checked 70 source files)

black....................................................................Passed
flake8...................................................................Passed
check-manifest...........................................................Passed
```
</details>

fridex commented:
I'll merge this to enable other PRs in. All the changes were done using black, added dots to docstrings and changed relative import. There are no functional changes.

fridex commented:
Feel free to raise any comments even once merged.
